cernopendata/modules/fixtures/data/records/cms-validated-runs.json

```diff
@@ -127,5 +127,70 @@
       }
     ]
   }
+},
+{
+  "abstract": {
+    "description": "This file describes which luminosity sections in which runs are considered good and should be processed."
+  },
+  "accelerator": "CERN-LHC",
+  "collaboration": {
+    "name": "CMS Collaboration",
+    "recid": "451"
+  },
+  "collections": [
+    "CMS-Validated-Runs",
+    "CMS-Validation-Utilities"
+  ],
+  "date_created": "2012",
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "txt"
+    ]
+  },
+  "doi": "10.7483/OPENDATA.CMS.C00V.SE32",
+  "experiment": "CMS",
+  "files": [
```

Comment:
@ArtemisLav I have added the files on EOS. Here is the (uri, size, sha1sum) file information:

```
root://eospublic.cern.ch//eos/opendata/cms/validated-runs/2012/Cert_190456-208686_8TeV_22Jan2013ReReco_Collisions12_JSON.txt 36630 b24efcfb1050b083aff2390df01fc16d6b67b836
```

Reply:
Please also put the record in a new file `cms-validated-runs-Run2012B.json` as mused about in https://github.com/cernopendata/opendata.cern.ch/pull/1865#issuecomment-347418847